tests/cpp/test_multidevice_transformer.cpp

```diff
@@ -395,6 +374,8 @@ TEST_P(DistributedTransformerTest, MLP_Layer) {
   for (TensorView* tv : tvsout) {
     fusion->addOutput(tv);
   }
+  shardBetween({tvw0, tvb0, tvw1}, {tvsout[3]}, tvw0);
+  shardBetween({tvx, tvb1}, {tvsout[3]}, tvx);
```

Comment:
Is there a simple way to assert that the final fusion contains only 1 allreduce? I can see from HostIR dump that 1 allreduce as expected is there, but not sure how I can get access this info easily.

Reply:
https://github.com/NVIDIA/Fuser/blob/cf141ed738b77b08e7c3bfe5e8b1311873c725e5/tests/cpp/test_multidevice_lower_communication.cpp#L25 is the simplest way I can think of at this moment.